wgpu/Cargo.toml

```diff
@@ -78,13 +78,22 @@ strict_asserts = ["wgc?/strict_asserts", "wgt/strict_asserts"]
 ## Log all API entry points at info instead of trace level.
 api_log_info = ["wgc/api_log_info"]
 
+## Enables serde functionality on common wgpu types.
+serde = ["serialize", "deserialize", "wgc/serde"]
+
+## Enables serialize functionality on common wgpu types.
+serialize = ["dep:serde", "wgc/serialize"]
+
+## Enables deserialize functionality on common wgpu types.
+deserialize = ["dep:serde", "wgc/deserialize"]
+
```

Comment:
unfortunately this doc / feature change needs to be copy pasted to here
https://github.com/gfx-rs/wgpu/blob/trunk/wgpu/src/lib.rs#L33
Same is true for the wgpu-core ones. (this should be done by `document-features` crate but we can't use this right now since it breaks vendoring for firefox builds)